src/Components/TranscoderInfo/index.js

```diff
@@ -36,40 +37,48 @@ const TDData = styled(TD)`
 `
 
 const TranscoderInfo = props => {
-  const { ...restProps } = props
+  const { myDelegateData, summaryData, ...restProps } = props
+
+  const totalStake = (myDelegateData && myDelegateData.totalStake) || 0
+  const delegatorStake = (summaryData && summaryData.delegator && summaryData.delegator.totalStakeInLPT) || 0
+  const participationInStake = totalStake > 0 ? (delegatorStake * 100) / totalStake : 0
+
+  const address = summaryData && summaryData.delegator && summaryData.delegator.delegateAddress
+  const missedRewardCalls = (myDelegateData && myDelegateData.last30RoundsMissedRewardCalls) || 0
+  const rewardCut = (myDelegateData && myDelegateData.rewardCut) || 0
+
+  const roi = (myDelegateData && myDelegateData.roiAbs) || 0
+  const roiEvery1000 = (myDelegateData && myDelegateData.roiPercent) || 0
+
   const isActive = true
-  const transcoderData = {
-    address: '0x12345678…9abcdef',
+  const delegateData = {
+    address: address,
     data: [
       {
         title: 'Your Participation In Stake',
-        data: '0.50%',
-      },
-      {
-        title: 'Position in Ranking',
-        data: (
-          <>
-            <IconUp /> #12
-          </>
-        ),
+        data: `${decimalPlaces(participationInStake)}%`,
       },
       {
         title: 'Reward Cut',
-        data: '5%',
+        data: `${decimalPlaces(rewardCut)}%`,
       },
       {
         title: 'Missed Reward Calls',
-        data: '2/5000',
+        data: `${missedRewardCalls}/30`,
+      },
+      {
+        title: 'Return of investment',
+        data: `${decimalPlaces(roi)}`,
       },
       {
-        title: 'Return Compared To #1',
-        data: '0.75 (-14%)',
+        title: 'Return of investment every 1000 LPT',
```

Comment:
R: Sorry I didn't noticed it before, please change this to `Return of investment (%)`. I think we should discuss opening a new issue in order to add a tooltip to explain that is the return every 1000 LPT

Reply:
Done